src/com/lilithsthrone/main/Main.java

```diff
@@ -796,25 +796,15 @@ public static String getQuickSaveName() {
 		if(!Main.game.isStarted()) {
 			return "QuickSave_intro";
 		}
-		return "QuickSave_"+Main.game.getPlayer().getName(false);
+		return "QuickSave_"+Main.game.getPlayer().getSurname();
```

Comment:
Can surname be empty for the player? If so, need an additional check for that.

Reply:
Surname can be an empty string, at which point this would become Quicksave_